test/common/upstream/cluster_update_tracker_test.cc

```diff
@@ -1,6 +1,7 @@
 #include "common/upstream/cluster_update_tracker.h"
 
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/cluster_manager.h"
+#include "test/mocks/upstream/thread_local_cluster.h"
```

Comment:
Is this needed?

Reply:
they are needed since `MockThreadLocalCluster` is used, now it's also added in the rule.